src/store/index.js

```diff
@@ -214,6 +215,88 @@ export class ApplicationStore {
             }
         });
     }
+
+    @computed
+    get degree2SizeFunc() {
```

Comment:
s/2/To/

Reply:
I guess it should mean `degree2SizeFunc` -> `degreeToSizeFunc`